resources/definitions/ultimaker_sketch.def.json

```diff
@@ -148,6 +163,7 @@
             "enabled": true,
             "value": "resolveOrValue('print_sequence') != 'one_at_a_time'"
         },
+        "print_sequence": { "enabled": false },
```

Comment:
Is there anything wrong with ona-at-a-time for sketch ?

Reply:
Its untested and we are missing some gantry measurements.  I'll make sure we test it before the next Cura release.

Reply:
Ok, makes sense to disable it for now then